src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -1116,3 +1118,25 @@ def pd_read_json(file) -> "DataFrame":
 
     with read_open(file) as f:
         return pd.read_json(f, orient="records", lines=True)
+
+
+def get_uuid_by_mac_id():
```

Comment:
does the mac stays the same for those containers in ci pipelines?

Reply:
In theory, it should be the same. I need to test it.

Reply:
According to the discussion, if there is a situation where multiple virtual machines share the MAC ID of the host, it can be ignored for now.